### PR TITLE
Feature/templates fix

### PR DESCRIPTION
**Pull request by drobnikj**

### CODE REVIEW

File: src/lib/consts.js [hunk 1]
```diff
@@ -6,18 +6,18 @@ const { KEY_VALUE_STORE_KEYS } = require('apify-shared/consts');
 // the name and description should be taken from package.json files in those dirs so that we edit it in a single location!
 // BTW the templates should be clustered by languages, now we have only JS but we'll add other languages later.
 exports.ACTS_TEMPLATES = {
```
Comment (drobnikj) [inline src/lib/consts.js]: @mtrunkat @jancurn Can you check renaming templates + new descriptions. Any suggestions for improvements?

File: src/lib/consts.js [hunk 2]
```diff
@@ -6,18 +6,18 @@ const { KEY_VALUE_STORE_KEYS } = require('apify-shared/consts');
...
             build: 'latest',
-            timeoutSecs: 3600,
-            memoryMbytes: 512,
+            timeoutSecs: 0,
+            memoryMbytes: 2048,
         },
     },
     puppeteer: {
-        name: 'Puppeteer ("puppeteer")',
+        name: 'Puppeteer - Boilerplate for loading a web page using headless Chrome and Puppeteer',
```
Comment (mtrunkat) [inline src/lib/consts.js]: Maybe - `Boilerplate for interaction with a web page using headless Chrome and Puppeteer`

File: src/lib/consts.js [hunk 3]
```diff
@@ -26,20 +26,20 @@ exports.ACTS_TEMPLATES = {
...
+            timeoutSecs: 3600,
+            memoryMbytes: 512,
         },
     },
-    plain_request_urls_list: {
-        name: 'Plain request url list crawler ("plain_request_urls_list")',
-        value: 'plain_request_urls_list',
-        dir: `${__dirname}/../templates/plain_request_urls_list`,
+    hello_word: {
+        name: 'Hello word - Example boilerplate from apify SDK documentation',
```
Comment (mtrunkat) [inline src/lib/consts.js]: `Hello word - An example actor demonstrating recursive crawling of a website using headless Chrome and Puppeteer.`

File: src/commands/create.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const inquirer = require('inquirer');
 const execWithLog = require('../lib/exec');
 const outputs = require('../lib/outputs');
 const { updateLocalJson } = require('../lib/files');
-const { setLocalConfig, setLocalEnv } = require('../lib/utils');
+const { setLocalConfig, setLocalEnv, npmCmd } = require('../lib/utils');
```
Comment (mtrunkat) [inline src/commands/create.js]: Maybe get it - `getNpmCmd()` ? To be clear that it's a getter

**Pull request merged**